### DISCUSSION TITLE
Use python3.11 and pytorch2.0 for performance

### DISCUSSION THREAD

**Discussion by eggplants** [AbdBarho/stable-diffusion-webui-docker#368] [Ideas]
As I see services/AUTOMATIC1111/Dockerfile, it uses `python:3.10.9-slim` and `torch==1.13.1+cu117`. I would like to update them to make processing faster if possible.

**Comment by AbdBarho** [comment]
I would prefer to wait until the original repo is updated to 2.0, since it is still untested.
